### PR TITLE
Move our extra solution to work with CTP6

### PR DESCRIPTION
**Pull request by jaredpar**
This change moves Roslyn2013.sln to work with Visual Studio 2015 CTP6.
This allows customers, and Jenkins, to build and test significantly more
code than was available with the old solution.

This is still not as full Roslyn.sln because not everything is buildable
against CTP6.  In particular our Visual Studio components and all of
their dependencies can't build due to API changes we depend on that came
into being after CTP6.  It is much closer though than our previous
solution.

As a part of this change I also renamed Roslyn2013.sln to
RoslynLight.sln.  The 2013 moniker was no longer valid and lacking a
better word I chose light (it has less projects and hence is lighter).

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet-bot test this please

File: src/Tools/Vsdconfig/Vsdconfig.targets [hunk 1]
```diff
@@ -5,7 +5,7 @@
           Inputs="@(VsdConfigXml);$(IntermediateOutputPath)\$(AssemblyName).dll"
           Outputs="$(OutDir)\$(AssemblyName).vsdconfig"
           Condition="'$(SkipGenerateVsdconfig)' != 'true'">
-    <Exec Command="&quot;$(VSSDK140Install)VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
+    <Exec Command="&quot;$(VS140COMNTOOLS)..\..\VSSDK\VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
```
Comment (tmeschter) [inline src/Tools/Vsdconfig/Vsdconfig.targets]: Is VSSDK140Install no longer defined?

**Comment by tmeschter** [comment]
:+1:

File: src/Tools/Vsdconfig/Vsdconfig.targets (hunk 1 above)
Comment (jaredpar) [inline src/Tools/Vsdconfig/Vsdconfig.targets]: I did this work on a fresh machine which had only VS 2015 CTP6 installed and `VSSDK140Install` was not defined.  It is defined on my long running dev machine though.  After looking at the output of env this was the common variable between the two machines. 

I had @amcasey take a look at this and he was fine with this switch.

Comment (Pilchie) [inline src/Tools/Vsdconfig/Vsdconfig.targets]: I had VSSDK140Install not show up on one machine for a day, and then it was back after a reboot.  Not sure why.

**Comment by Pilchie** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

**Comment by amcasey** [comment]
:+1:

**Pull request merged**